spec/models/user_spec.rb

```diff
@@ -87,4 +87,12 @@
       expect(alice.follows?(bob)).to be false
     end
   end
+
+  describe "#deactivate!" do
+    it "deactivates an active user" do
+      alice = Fabricate(:user, active: true)
+      alice.deactivate!
+      expect(alice).to_not be_active
```

Comment:
You should do:

```ruby
expect(alice.reload).to_not be_active
```

That will make sure that you saved the change to the db, and not just to the object in memory.